x-pack/test/cases_api_integration/security_and_spaces/tests/trial/cases/assignees.ts

```diff
@@ -33,276 +33,360 @@ export default ({ getService }: FtrProviderContext): void => {
-        count_open_cases: 2,
-      });
-    });
+        const cases = await findCases({
+          supertest,
+          query: { assignees: [profile[0].uid] },
+        });
 
-    it('updates the assignees on a case', async () => {
-      const profiles = await suggestUserProfiles({
-        supertest: supertestWithoutAuth,
-        req: {
-          name: 'delete',
-          owners: ['securitySolutionFixture'],
-          size: 1,
-        },
-        auth: { user: superUser, space: 'space1' },
+        expect(cases).to.eql({
+          ...findCasesResp,
+          total: 2,
+          cases: [caseWithDeleteAssignee1, caseWithDeleteAssignee2],
+          count_open_cases: 2,
+        });
       });
 
-      const postedCase = await createCase(supertest, getPostCaseRequest());
-
-      const patchedCases = await updateCase({
-        supertest,
-        params: {
-          cases: [
-            {
-              id: postedCase.id,
-              version: postedCase.version,
-              assignees: [{ uid: profiles[0].uid }],
-            },
-          ],
-        },
+      it("filters cases using the assigned users by constructing an or'd filter", async () => {
+        const profileUidsToFilter = await suggestUserProfiles({
+          supertest: supertestWithoutAuth,
+          req: {
+            name: 'only',
+            owners: ['securitySolutionFixture'],
+            size: 2,
+          },
+          auth: { user: superUser, space: 'space1' },
+        });
+
+        await createCase(supertest, postCaseReq);
+        const caseWithDeleteAssignee1 = await createCase(
+          supertest,
+          getPostCaseRequest({
+            assignees: [{ uid: profileUidsToFilter[0].uid }],
+          })
+        );
+        const caseWithDeleteAssignee2 = await createCase(
+          supertest,
+          getPostCaseRequest({
+            assignees: [{ uid: profileUidsToFilter[1].uid }],
+          })
+        );
+
+        const cases = await findCases({
+          supertest,
+          query: { assignees: [profileUidsToFilter[0].uid, profileUidsToFilter[1].uid] },
+        });
+
+        expect(cases).to.eql({
+          ...findCasesResp,
+          total: 2,
+          cases: [caseWithDeleteAssignee1, caseWithDeleteAssignee2],
+          count_open_cases: 2,
+        });
       });
 
-      const retrievedProfiles = await bulkGetUserProfiles({
-        supertest,
-        req: {
-          uids: patchedCases[0].assignees.map((assignee) => assignee.uid),
-          dataPath: 'avatar',
-        },
+      it('filters cases with no assignees', async () => {
+        const profile = await suggestUserProfiles({
+          supertest: supertestWithoutAuth,
+          req: {
+            name: 'delete',
+            owners: ['securitySolutionFixture'],
+            size: 1,
+          },
+          auth: { user: superUser, space: 'space1' },
+        });
+
+        const caseWithNoAssignees = await createCase(supertest, postCaseReq);
```

Comment:
Could these be created using a `Promise.all` or do we need them in a certain order?